makefiles/buildtests.inc.mk

```diff
@@ -38,7 +38,7 @@ buildtest:
 	@ \
 	BUILDTESTOK=true; \
 	APP_RETRY=0; \
-	for BOARD in $$($(MAKE) -s info-boards-supported); do \
+	for BOARD in ${BOARDS}; do \
```

Comment:
i.e., I test with `tests/periph_rtc` because of #6690 and just added `BOARD_BLACKLIST := arduino-zero` to its Makefile, which successfully excludes this board though it has the feature `periph_rtc`